src/sentry/eventstream/base.py

```diff
@@ -110,7 +110,7 @@ def _dispatch_post_process_group_task(
                 queue=queue,
             )
 
-    def _get_queue_for_post_process(self, event: Event) -> str:
```

Comment:
Changes around `GroupEvent` are unrelated, that's just where the journey wrt typing started